### PR TITLE
V13.x.x

### PR DESCRIPTION
**Pull request by IanMcShane**
this includes Play 2.8 upgrades

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,66 +19,71 @@ It encapsulates some common concerns for calling other HTTP services on the HMRC
...
 
-### Version 12.0.0
+```scala
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
-#### SessionKeys
-The fields `authProvider` ('ap'), `userId` and `token` are no longer available in the session and these names have been removed from SessionKeys object.
+libraryDependencies += "uk.gov.hmrc" %% "http-verbs-play-xx" % "x.x.x"
+```
+Where `play-xx` is `play-25`, `play-26` or `play-27` depending on your version of Play.
```
Comment (cjwebb) [inline README.md:31]: This line is outdated.

**Comment by IanMcShane** [comment]
test this please

**Review by colin-lamed — APPROVED** [review]

**Comment by IanMcShane** [comment]
test this please

**Comment by IanMcShane** [comment]
test this please

**Comment by IanMcShane** [comment]
test this please

**Pull request merged**